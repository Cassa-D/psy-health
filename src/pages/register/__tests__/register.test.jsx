import React from "react";

import RegisterFormContainer from "../../../containers/register-form";
import { shallow } from "../../../containers/search-filter/__tests__/node_modules/enzyme";
import Register from "../index";

describe("Renders", () => {
  it("renders register with RegisterFormContainer", () => {
    const wrapper = shallow(<Register where="/" history={{}} />);
    expect(wrapper.find(RegisterFormContainer)).toBeTruthy();
  });

  it("renders register with img", () => {
    const wrapper = shallow(<Register where="/" history={{}} />);
    expect(wrapper.find("img")).toBeTruthy();
  });

  it("renders register with h1 correct", () => {
    const wrapper = shallow(<Register where="/" history={{}} />);
    const h1 = wrapper.find("h1");
    expect(h1).toBeTruthy();
    expect(h1.text()).toBe("Register");
  });

  it("renders register with span patient", () => {
    const wrapper = shallow(<Register where="/register" history={{}} />);
    const patient = wrapper.find(".usr");
    expect(patient).toBeTruthy();
    expect(patient.text()).toBe("Sou Paciente");
  });

  it("renders register with span psic", () => {
    const wrapper = shallow(<Register where="/register/psi" history={{}} />);
    const psic = wrapper.find(".psi");
    expect(psic).toBeTruthy();
    expect(psic.text()).toBe("Sou Psic√≥logo");
  });
});
