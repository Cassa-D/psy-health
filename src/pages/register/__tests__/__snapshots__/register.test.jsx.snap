// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders register 1`] = `ShallowWrapper {}`;

exports[`Snapshot renders register for psi 1`] = `ShallowWrapper {}`;
